lib/src/widgets/detailCardsWidgets/movie_resume.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+import '../../utils/ui_constants.dart';
+
+class MovieDescription extends StatelessWidget {
```

Comment:
The name of the file its different form the name of the class, it should be `movie_descritption.dart`

Reply:
done.